real_estate_module/models/estate_property.py

```diff
@@ -0,0 +1,26 @@
+from odoo import models, fields
```

Comment:
```suggestion
from odoo import fields, models
```
import should be in order